### DISCUSSION TITLE
Suggestion: Deadlines for flips submit, invites activation and nodes syncing. Tasks in the client with reminders about it.

### DISCUSSION THREAD

**Discussion by Gutalean** [idena-network/idena-desktop#630] [Ideas]
1. People should be restricted to submit their flips later than 2 days before the validation in order to help these flips to be propagated through the network. My current task in the client should be like: 'Please, submit 3 flips before [Validation.Date - 2]'.

2. All the invites should be activated 1 day before the validation. My current task for candidates should be: 'Please sync your node and activate an invite before [Validation.Date - 1]'. For the users with available invites it could be: 'Please make sure that all your issued invites will be activated before [Validation.Date - 1]'. This measure will minimize the problem of last moment syncing of huge amount of new users.

3. Synchronization for the nodes of candidates without activated invites should be refused by the network during the last 24 hours of an epoch. My current task for the other participants should be: 'Please keep your node online for the next 24 hours to take part in the validation session'. This measure will minimize the problem of last moment syncing of huge amount of new users.

**Comment by jugi1** [comment]
I support this.
EVen sending flips day before validation is better, as many people are sending them even hours or minutes before  validation
